src/internal/observable/of.ts

```diff
@@ -45,6 +43,9 @@ export function of<T, T2, T3, T4, T5, T6, T7, T8, T9>(a: T, b: T2, c: T3, d: T4,
 export function of<T>(...args: T[]): Observable<T>;
 /* tslint:enable:max-line-length */
 
+/** @deprecated use {@link scheduled} instead `scheduled([a, b, c], scheduler)` */
+export function of<T>(...args: (T | SchedulerLike)[]): Observable<T>;
```

Comment:
I would move it to just above the `/* tslint:enable:max-line-length */` line. Throughout the codebase, those lint flags wrap the overload signatures, so this file should stick with that convention.